tests/test_search.py

```diff
@@ -1,6 +1,19 @@
+import time
+
+import pytest
+
+import ingest.bulk_upload
 from oracc_rest import ESearch
 
 
+@pytest.fixture
+def uploaded_entries(es, entries):
+    """A fixture to ensure that the test glossary entries have been uploaded."""
+    ingest.bulk_upload.upload_entries(es, entries)
+    time.sleep(2)  # a small delay to make sure the upload has finished
```

Comment:
Just an idea, if it's not too difficult to implement: can you replace this with a while loop that checks if the bulk upload is finished, and fails if it doesn't after a given number of tries.

Reply:
Done!
I'm planning to eventually (soon) change the tests so that they use an Elasticsearch plugin for pytest, which will handle all the preparation and should simplify this anyway.